src/wmtk/TetMeshOperationExecutor.hpp

```diff
@@ -160,8 +160,43 @@ class TetMesh::TetMeshOperationExecutor
     const long operating_edge_id() const { return m_operating_edge_id; }
 
 
+    /*
+
+       */
+    /**
+     * @brief split edge v1-v2
+     *
+     *            v3
+     *            /\\
+     *     ear1  /| \ \   ear2
+     *          / |  \  \
+     *         /  |   \   \
+     *        /   |    \    \
+     *       /    |     \     \
+     *      /     |      \     _\ v4
+     *     /______|_______\_ -
+     *    v1     v_new      v2
+     *
+     *   input: tuple(v1, v1-v2, v1-v2-v3, v1-v2-v3-v4)
+     *
+     * @return Tuple(v1, v1-v_new, v1-v_new-v3, v1-v_new-v3-v4)
+     */
     Tuple split_edge();
 
+
+    //  5 --------- 3 ---------- 6
+    //   \  \      / \\        /
+    //    \      \/   \ \     /
+    //     \     /    \\  \  /
+    //      \   /       \  \\ 4
+    //        1 --------- 2/      tuple edge 1-2
+    //
+    /**
+     * @brief split edge v1-v2, input: tuple(v1, v1-v2, either face, v1-v2-v3-v4)
```

Comment:
Please modify that comment in the same way you modified the `split_edge` comment. That one looks good now, although you could use a few more words to describe the tuple, in my opinion.